.jenkins/noobaa-core.groovy

```diff
@@ -0,0 +1,72 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/noobaa/noobaa-core'
+def ci_git_branch = 'master'
+
+// temporary variables, see .travis.yml
+def TRAVIS_COMMIT = "01234abcde"
+def IMAGE_TAG = "noobaa-${TRAVIS_COMMIT}"
+def TESTER_TAG = "noobaa-tester-${TRAVIS_COMMIT}"
+def NO_CACHE = "NO_CACHE=true"
+def SUPPRESS_LOGS = "SUPPRESS_LOGS=true"
+def DEPLOY_MINIKUBE_REDIRECT = "/dev/null"
```

Comment:
As Jenkins does not have travis restrictions on logs size, we can remove this redirection